kubernetes/charts/pantheon-genesis/values.yaml

```diff
@@ -0,0 +1,41 @@
+# Default values for genesys.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: pegasyseng/pantheon
+  tag: "1.1.3"
+  # Always | IfNotPresent
+  pullPolicy: IfNotPresent
+
+validators:
+  generated: true
+
+config:
```

Comment:
Can we put a comment here saying hashes and things came from the quickstart doco? https://docs.pantheon.pegasys.tech/en/stable/Tutorials/Create-IBFT-Network/